content/en/tracing/trace_collection/otel_instrumentation/python.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Custom Instrumentation of Python Applications with the OpenTelemetry API
+kind: documentation
+description: 'Instrument your Python application with OTel API to send traces to Datadog.'
+code_lang: python
+type: multi-code-lang
+code_lang_weight: 10
+further_reading:
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
```

Comment:
Just noting that we have further reading links in the front matter, but we don't have the Further Reading section at the bottom of the article.